spec.html

```diff
@@ -178,7 +178,10 @@
     <emu-grammar>UnaryExpression : `delete` UnaryExpression</emu-grammar>
     <ul>
       <li>
-        It is a Syntax Error if the |UnaryExpression| is contained in strict mode code and the derived |UnaryExpression| is <emu-grammar>PrimaryExpression : IdentifierReference</emu-grammar>, <ins><emu-grammar>MemberExpression : MemberExpression `.` PrivateName</emu-grammar>, or <emu-grammar>CallExpression : CallExpression `.` PrivateName</emu-grammar></ins>.
+        It is a Syntax Error if the |UnaryExpression| is contained in strict mode code and the derived |UnaryExpression| is <emu-grammar>PrimaryExpression : IdentifierReference</emu-grammar>.
+      </li>
+      <li>
+        <ins>It is a Syntax Error if the derived |UnaryExpression| is <emu-grammar>MemberExpression : MemberExpression `.` PrivateName</emu-grammar> or <emu-grammar>CallExpression : CallExpression `.` PrivateName</emu-grammar></ins>.
```

Comment:
We discussed this previously in https://github.com/tc39/proposal-class-fields/issues/30 . As described there, I don't think this change is necessary.

Reply:
Ah, I see.